app/models/steps/individual.rb

```diff
@@ -54,7 +54,7 @@ def restart
     end
 
     def user_is_not_requester
-      if user_id && user_id == proposal.requester_id
+      if user && user == proposal.requester
```

Comment:
One benefit of using objects vs foreign keys here is that it checks that the record actually exists. eg; there could be a `user_id` but no user for that `id` (perhaps user was deleted?)

 I generally follow a similar rule in validations "Validate the associated belongs_to object (user), not the database column (user_id)."

I am not too worried about the performance implications, but can remove if you think this is unnecessary.